server/src/main/java/io/crate/execution/engine/window/WindowFunctionBatchIterator.java

```diff
@@ -86,6 +86,7 @@
                                         Executor executor,
                                         List<WindowFunction> windowFunctions,
                                         List<? extends CollectExpression<Row, ?>> argsExpressions,
+                                        Boolean[] ignoreNulls,
```

Comment:
Minor, I think i would probably add an something like `assert args.length == ignoreNulls.length : "ignore null argument must be defined for each input` .

Reply:
thanks! added